google/cloud/spanner_dbapi/cursor.py

```diff
@@ -134,13 +137,12 @@ def description(self):
 
     @property
     def rowcount(self):
-        """The number of rows produced by the last `execute()` call.
+        """The number of rows produced by the last `.execute()` call.
```

Comment:
I think we should add a note that rows are counted only for UPDATE requests. Otherwise some users may expect `rowcount` to work in all cases, but they'll get -1.

Reply:
Updated the comment to say the rowcount only works for update/delete queries and not select queries.